processor/src/test/java/org/mapstruct/ap/test/abstractclass/generics/Target.java

```diff
@@ -23,22 +23,22 @@
  *
  */
 public class Target {
-    private ItemC itemC;
-    private ItemB itemB;
+    private AnimalKey itemC;
+    private KeyOfAllBeings itemB;
```

Comment:
The properties (including getter/setter) should be named `animalKey` and `keyOfAllBeings` to make it easier to grasp in the mapper definition.